src/xdocs/checks/annotation/annotationlocation.xml

```diff
@@ -233,8 +233,8 @@ private boolean field2;
 @Override //ok
 public boolean equals(Object obj) { return true; }
 @Mock DataLoader loader; //ok
-@SuppressWarnings("deprecation") DataLoader loader; //ok
-@SuppressWarnings("deprecation") public int foo() { return 1; } //violation
+@SuppressWarnings(&quot;deprecation&quot;) DataLoader loader; //ok
+@SuppressWarnings(&quot;deprecation&quot;) public int foo() { return 1; } //violation
```

Comment:
Not sure why messages related PR has this change.

Can we simply move templatization to separate PR and merge it quicker. To let this PR be only messages related.

Reply:
This is the underlying parser escaping all text inside `<source>` elements. I am having a hard time overriding it.

Reply:
I meant , why we create new template file in this PR. We just need replacement of content to macros in existing templates

Reply:
It is ok to not migrate all pages now , but at least some, but still in separate easy to review/merge PRs.

Reply:
Please choose 10 modules and migrate them to templates ASAP, it will be your base set of modules to test your macros.
We still be able to make implementation that satisfies some of such modules and work on extension in separate PRs.

Reply:
If that is kind of automate-able, please migrate all, I will merge quickly.
It will be easily review as they should be no diff (except escaping and indentation maybe)

Reply:
Opened at https://github.com/checkstyle/checkstyle/issues/13498.

Reply:
Done. Changed an existing template instead.